content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -405,6 +410,113 @@ instances:
 ### Serverless environment
 
 For more information about AWS Lambda functions, see [how to connect your Lambda telemetry using tags][15].
+
+### OpenTelemetry environment
+
+In OpenTelemetry environments, you can use the [OpenTelemetry Collector][16] to collect and batch telemetry data (like metrics, traces, and logs) by leveraging the [Datadog Exporter][17] to send the data to Datadog without the Datadog Agent.
+
+Optionally, you can send telemetry data from [applications instrumented with the OpenTelemetry SDKs with the Datadog Agent][20], which collects the data for Datadog.
+
+To set up unified service tagging in an OpenTelemetry environment:
+
+1. Depending on the OpenTelemetry component you are using, set the following attributes to ensure proper tagging:
```

Comment:
The "following attributes" wording feels a little confusing, since an explicit list of attributes does not follow, just two examples.

Maybe you could say something like "To ensure proper tagging, set the required attributes for the OpenTelemetry component you are using, as shown in the examples below." Then the examples could just say "With the Python OpenTelemetry SDK:"

Reply:
```suggestion
1. Set the required attributes to ensure proper tagging for the OpenTelemetry component you are using.
```